webpack/ForemanInventoryUpload/Components/PageHeader/__tests__/PageHeader.test.js

```diff
@@ -1,13 +1,107 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import { render, screen } from '@testing-library/react';
 import { testComponentSnapshotsWithFixtures } from '@theforeman/test';
 
 import PageHeader from '../PageHeader';
 
+// Mock the useAdvisorEngineConfig hook
+jest.mock('../../../../common/Hooks/ConfigHooks', () => ({
+  useAdvisorEngineConfig: jest.fn(),
+}));
+
+import { useAdvisorEngineConfig } from '../../../../common/Hooks/ConfigHooks';
+
+const middlewares = [thunk];
+const mockStore = configureMockStore(middlewares);
+
+// Mock store configuration with required data for PageHeader components
+const renderOptions = {
+  API: {
+    INVENTORY_SETTINGS: {
+      response: { 
+        subscriptionConnectionEnabled: true,
+        insightsMinimalDataCollection: false 
+      },
+    },
+  },
+  ForemanRhCloud: {
+    inventoryUpload: {
+      accountsList: {
+        CloudConnectorStatus: {},
+      },
+    },
+  },
+};
+
 const fixtures = {
   'render without Props': {},
   /** fixtures, props for the component */
 };
 
 describe('PageHeader', () => {
+  beforeEach(() => {
+    // Reset mock before each test
+    useAdvisorEngineConfig.mockReset();
```

Comment:
You can just add the property to the test store, and then you won't have to mock the accessor

Reply:
I’m not sure I fully understand what you mean. Could you clarify?